homework.rb

```diff
@@ -1,6 +1,10 @@
-VARIANT = 0
+VARIANT = 4
 
 def generate
-  # put your solution here (delete the line below)
-  rand(1000000).to_s + '-' + rand(1000000).to_s
+
+  alphabet = ('A'..'Z').to_a
+  alphabet_count = alphabet.count
+
+  "#{alphabet[rand(alphabet_count)]}#{alphabet[rand(alphabet_count)]}#{rand(10...100)}-"\
```

Comment:
Eye blood. Please use the power of Ruby to create more efficient code style. (next line too)